packages/ui/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { LogoNameMap } from './logo-name-map'
+
+interface LogoProps {
+  name: keyof typeof LogoNameMap
+  size?: number
+  original?: boolean // Uses brand color when true
+}
+
+const Logo: React.FC<LogoProps> = ({ name, size = 24, original = false }) => {
+  const icon = LogoNameMap[name]
+
+  return (
+    <svg width={size} height={size} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d={icon.path} fill={original ? `#${icon.hex}` : 'currentColor'} />
```

Comment:
Interesting that they were able to get all logos to work on a single-path/single-colour basis and only with fills and not strokes 🤔